LICENSE.md

```diff
@@ -1,20 +1,20 @@
 The Yii framework is free software. It is released under the terms of
 the following BSD License.
 
-Copyright © 2008-2013 by Yii Software LLC (http://www.yiisoft.com)
+Copyright © 2008 by Yii Software (<https://www.yiiframework.com/>)
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 
- * Redistributions of source code must retain the above copyright
+* Redistributions of source code must retain the above copyright
```

Comment:
Align it vertically, like here: https://raw.githubusercontent.com/yiisoft/package-template/master/LICENSE.md